src/map/map.h

```diff
@@ -924,6 +935,9 @@ void map_skill_damage_free(struct map_data *m);
 void map_skill_damage_add(struct map_data *m, uint16 skill_id, int pc, int mob, int boss, int other, uint8 caster);
 #endif
 
+bool map_skill_duration_add(struct map_data *md, uint16 skill_id, uint16 per);
```

Comment:
yes, I know this does not matter at declaration at all, but `md` for `mob_data`, if it possible please change to struct map_data *m, and the line below too, for avoiding confusing. :shipit: